src/pages/components/multiselect.mdx

```diff
@@ -72,42 +72,98 @@ import MultiselectSizeCode from '!!raw-loader!../../examples/dropdowns/multisele
 
 <Configuration reactPackage={props.data.mdx.package} components={props.data.mdx.components} />
 
+## Accessibility
+
+If no Label component is provided with the implementation, include an `aria-label` to provide an
+accessible name for the field.
+
 ## API
 
+The Multiselect component expects the following overall structure:
+
+```jsx isLight
+<Dropdown>
+  <Field>
+    <Label>
+    <Hint>
+    <Multiselect>
+  </Field>
+  <Menu>
+    <Item>
+    <Item>
+    ...
+  </Menu>
+</Dropdown>
+```
+
+<!-- markdownlint-disable no-emphasis-as-heading -->
+
+**Implementation notes**
+
+<!-- markdownlint-enable no-emphasis-as-heading -->
+
+- The dropdowns package does not supply filtering logic. The simplest way to perform filtering is to
+  control the `inputValue` and `onInputValueChange` props, and conditionally render Item components as
+  necessary.
+
+- Use input debouncing to delay Item rendering and prevent Downshift from reapplying accessibility
+  and interaction attributes.
+
+- Due to a known [issue](https://github.com/facebook/react/issues/11538#issuecomment-390386520) caused
+  by google translation extension, any conditionally rendered text node should be wrapped with a `span`
+  element.
```

Comment:
Would this copy benefit from being in a fragment?

Reply:
It might. I was having a hard time deciding where to draw the line for DRYing out repeated information, so I only did this for complete sections like the information on dropdowns. But I'm open to suggestions. Where would you draw the line?

Reply:
This had 3 line breaks (3 bullets in this case) and constituted a subheading (Implementation notes) that was repeated. This triggered this thought to me. If it was one to three sentences and did not have it's own subheading I probably wouldn't split this out. I think it would be more maintainable as fragment imported to both pages.